cbf/v1/cbf-v1.0.0.yaml

```diff
@@ -0,0 +1,556 @@
+                        message: Spaces not allowed in facility code
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: ../../models/ErrorResponse.yaml
+      operationId: create-v1-booking-forecast
+      description: Submits a Booking forecast.
+      parameters:
+        - $ref: ../../models/Headers.yaml#/components/parameters/APIVersionHeader
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/BookingForecast'
+        description: ''
+components:
+  schemas:
+    BookingForecast:
+      title: BookingForecast
+      x-stoplight:
+        id: keuoy8j4qkve1
+      type: object
+      properties:
+        partnerIdentity:
+          $ref: '#/components/schemas/PartnerIdentity'
+        carrierServiceName:
+          type: string
+          pattern: ^\S+(\s+\S+)*$
+          maxLength: 50
+          example: Great Lion Service
+          description: The name of a service as specified by the carrier
+        carrierServiceCode:
+          type: string
+          pattern: ^\S+(\s+\S+)*$
+          maxLength: 11
+          example: FE1
+          description: The carrier-specific code of the service for which the schedule details are published.
+        universalServiceReference:
+          type: string
+          description: 'A global unique service reference, as per DCSA standard, agreed by VSA partners for the service. The service reference must match the regular expression pattern: SR\d{5}[A-Z]. The letters SR followed by 5 digits, followed by a checksum-character as a capital letter from A to Z.'
+          example: SR12345A
+          maxLength: 8
+          pattern: '^SR\d{5}[A-Z]$'
+        vessels:
+          type: array
+          items:
+            $ref: '#/components/schemas/Vessel'
+      required:
+        - partnerIdentity
+        - carrierServiceName
+        - carrierServiceCode
+        - vessels
+    PartnerIdentity:
+      title: PartnerIdentity
+      x-stoplight:
+        id: mnr99wlnor6is
+      type: object
+      properties:
+        code:
+          type: string
+          format: regex
+          pattern: ^\S+(\s+\S+)*$
+          example: MMCU
+          maxLength: 4
+          description: SCAC code or SMDG Liner Code of the carrier.
+        codeListProvider:
+          type: string
+          example: NMFTA
+          description: The provider used for identifying the partner code
+          enum:
+            - SMDG
+            - NMFTA
+        agreementTypeCode:
+          type: string
+          example: VSA
+          enum:
+            - VSA
+            - SCA
+          description: |-
+            Possible values are:
+
+            - VSA (Vessel Sharing Agreement)
+            - SCA (Slot Charter Agreement)
+      required:
+        - code
+        - codeListProvider
+        - agreementTypeCode
+    Vessel:
```

Comment:
I seem to recall you arguing that transportCall in OVS should not be called transportCall since it was something slightly different. You are now calling this object a `Vessel` - but the object also includes forecasts. Have you thought about this?

Reply:
We did think about this actually, we tried to take it on a case-to-case basis. For the transport call scenario, I believe the meaning of transportCall in OVS does not make much sense and hence also against it being used there, for example, a portOfCall would make more sense in OVS.
In this case, it is primarily a vessel with added forecasts within the vessel for the vessel.
If you think this might not be the right way to do it, please do point it out. 
Open to naming suggestions too.

Reply:
I do agree with you that this would mean a vessel object here would conflict with the vessel object in another standard.